float_data_for_tock.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import json
+
+api_key = os.environ.get('FLOAT_API_KEY')
+headers = {'Authorization': 'Bearer ' + api_key}
+endpoint = 'tasks'
+year = "2016"
+month = "04"
+day = "01"
+weeks = "52"
+timecard_date = str(year + "-" + month + "-" + day)
+payload = {'start_day': timecard_date, 'weeks': weeks}
+url = 'https://api.floatschedule.com/api/v1/' + endpoint
+r = requests.get(url, headers=headers, params=payload)
+output = r.content
```

Comment:
Actually, to fix this issue, I would think all you need to do is load the json string, e.g., `json.loads(r.content)`.  More info here:  [`json.loads`](https://docs.python.org/3/library/json.html#json.loads).